setup.py

```diff
@@ -42,6 +42,8 @@
             "sphinx-autobuild",
             "sphinx-kr-theme",
             "twine",
+            "pytest>=5.3.0",
+            "pytest-asyncio>=0.14.0",
```

Comment:
@deathowl I think both pytest dependencies are already covered at the pytest-testslide setup.py and can be safely removed from here.